agent.py

```diff
@@ -16,8 +18,10 @@ def __init__(self):
             self.configuration = self.get_configuration_from_yml()
             self.influxdb_client = InfluxDBClient(url=self.configuration['INFLUXDB_URL'],
                                                   token=self.configuration['INFLUXDB_TOKEN'])
-            self.influxdb_write_options = SYNCHRONOUS
+            self.influxdb_write_options = ASYNCHRONOUS
             self.influxdb_data_writer = self.influxdb_client.write_api(self.influxdb_write_options)
+            self.docker_client = docker.from_env()
+            self.executor = ThreadPoolExecutor(max_workers=MAX_WORKERS)
         except Exception as e:
             print(f"Configuration failed due to : {e}")
```

Comment:
Would be nice to mark it as an error (eg. [ERROR] label at the beginning of string)
```suggestion
            print(f"[ERROR] Configuration failed due to : {e}")
```